### DISCUSSION TITLE
How is the mod able to get schematics across instances in multiMC?

### DISCUSSION THREAD

**Discussion by Theooutthedore** [maruohon/litematica#794] [General]
How is the mod able to get schematics across instances in multiMC? But unable to manually have a external folder with other schematics? I think i have seen that a global folder is planned to be added, anyway, thanks for the great mod, i hope the development work is going smoothly, and that the feature will be implemented so i can clean up my mess of MC files everywhere :P

**Comment by maruohon** [comment]
See Generic -> `customSchematicBaseDirectoryEnabled` and `customSchematicBaseDirectory`.

**Reply by Theooutthedore** [reply]
ahh! im surprised i managed to avoid any solutions to my question in my search, thank you very much for such a fast reply as well!
